draft/draft-ietf-netconf-https-notif.xml

```diff
@@ -198,11 +205,15 @@
 	</t>
       </section>
       <section title="Request">
-        <t>To learn the capabilities of a receiver, a publisher can issue an
-          HTTPS GET request to the "capabilities" resource (see <xref target= "overview"/>) on
-          the receiver with "Accept" header set using the "application/xml"
-          and/or "application/json" media-types, with the latter as mandatory
-          to implement, and the default in case the type is not specified.</t>
+        <t>To learn the capabilities of a receiver, a publisher can
+        issue an HTTPS GET request to the "capabilities" resource (see
+        <xref target= "overview"/>) on the receiver with "Accept"
+        header set using the "application/xml" as defined in <xref
+        target="RFC7303">XML Media Types</xref>, and/or
+        "application/json" as defined in <xref
+        target="RFC8259">JSON</xref> media-types, with the latter as
+        mandatory to implement, and the default in case the type is
+        not specified.</t>
```

Comment:
Again, JSON is NOT mandatory-to-implement.

Reply:
Hmm. I thought I removed that fragment ... at least I do not see it anymore at my end. Strange why you are still seeing it.